api/v1/cluster_webhook.go

```diff
@@ -864,6 +864,13 @@ func (r *Cluster) validateConfiguration() field.ErrorList {
 		// validateImageName function
 		return result
 	}
+	if psqlVersion <= 100000 {
+		result = append(result,
+			field.Invalid(
+				field.NewPath("spec", "imageName"),
+				r.Spec.ImageName,
+				"Invalid postgresql version. Versions 11 or above are supported"))
```

Comment:
How about using "Unsupported postgresql version" instead of "Invalid postgresql version" @sxd